### DISCUSSION TITLE
哪些概念要收錄

### DISCUSSION THREAD

**Discussion by dannypsnl** [FizzyElt/functional-programming#92] [General]
在寫的時候發現有時候好像只是在湊卡片，不是真的有什麼要分享，要是定義在 nlab 有完整呈現的話好像直接參考就好了？

有些核心概念如 F-algebra 還是會在站內解釋而且給出定義，我在考慮的是要是太簡單的如 terminal, initial 這種是不是不特別寫

**Comment by dannypsnl** [comment]
但網頁有點容易死亡，應該也要加上紙本參考

**Comment by FizzyElt** [comment]
有些雖然是簡單的定義及概念，但不太好聯想或想像
我認為可以後續補一些備註，引導讀者這些基本概念的用途以及在這裡代表什麼意義
